core/domain/activity_domain_test.py

```diff
@@ -76,6 +76,17 @@ def test_to_dict(self) -> None:
             }
         )
 
+    def test_from_dict(self) -> None:
+        sample_dict = {
+            'type': 'exploration',
+            'id': '1234'
+        }
+        sample_activity_object = activity_domain.ActivityReference.from_dict(sample_dict) # pylint: disable=line-too-long
+        self.assertEqual(
```

Comment:
Can we also assert that the object's fields are correct?

Reply:
Done!